common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -937,6 +948,76 @@ def test_user_can_cancel_the_team_creation(self):
 
 
 @ddt.ddt
+class DeleteTeamTest(TeamFormActions):
+    """
+    Tests for deleting teams.
+    """
+
+    def setUp(self):
+        super(DeleteTeamTest, self).setUp()
+
+        self.set_team_configuration(
+            {'course_id': self.course_id, 'max_team_size': 10, 'topics': [self.topic]},
+            global_staff=True
+        )
+        self.create_edit_delete_team_page = CreateEditDeleteTeamPage(self.browser, self.course_id, self.topic)
+
+        self.team = self.create_teams(self.topic, num_teams=1)[0]
+        self.team_page = TeamPage(self.browser, self.course_id, team=self.team)
+        self.team_page.visit()
+
+    def test_cancel_delete(self):
+        """
+        Scenario: The user should be able to cancel the Delete Team dialog
+        Given I am staff user for a course with a team
+        When I visit the Team profile page
+        Then I should see the Edit Team button
+        And When I click edit team button
+        Then I should see the Delete Team button
+        When I click the delete team button
+        And I cancel the prompt
+        Then I should remain on the page
+        """
+        self.team_page.click_edit_team_button()
+        self.create_edit_delete_team_page.wait_for_page()
+        self.create_edit_delete_team_page.delete_team_button.click()
+        self.create_edit_delete_team_page.cancel_delete_team_dialog()
+        self.assertTrue(self.create_edit_delete_team_page.is_browser_on_page())
```

Comment:
Wouldn't this be true even if the cancel did nothing? Maybe you should verify that if you refresh the page that the team is still there. You could also verify that the modal has gone away, although maybe that should be a given.